src/emc/usr_intf/gscreen/gscreen.py

```diff
@@ -53,10 +53,11 @@ def _print_help():
 import gi
 gi.require_version("Gtk","3.0")
 gi.require_version("Gdk","3.0")
-gi.require_version('PangoCairo', '1.0')
-from gi.repository import Gtk,Gdk,GObject,Pango,PangoCairo,cairo,GLib
+from gi.repository import Gtk, Gdk, GLib
 from gi.repository import Pango as pango
+
 try:
+    gi.require_version('Vte', '2.91')
     from gi.repository import Vte as vte
 except:
     LOG.error("**** WARNING GSCREEN: could not import vte terminal - is package installed?")
```

Comment:
this  should be a warning only. Gscreen screens can run without Vte.
I would remove the ****** WARNING GSCREEN.